.github/workflows/e2e_tests.yaml

```diff
@@ -55,6 +55,16 @@ jobs:
           git submodule update --init --recursive
       # - name: Update Kernel and Linux Headers
       #   run: printf 'y\nn\n' | sudo ./kernelupdate.sh
+      - name: Install go related
+        run: |
+          cd /tmp; wget https://dl.google.com/go/go1.13.9.linux-amd64.tar.gz
```

Comment:
I tried latest go1.17. From functionality it's working, but setup is a little different. For example, I see error "go get: installing executables with 'go get' in module mode is deprecated". After installed, then function it's good.

I think we need to decide a go version for now. go1.17 is current latest version, but sometime later it will be a normal "outdated" version. The reason I choose go1.13.9 is because it's the same go version in arktos. Since arktos and mizar both belong to Centaurus, I prefer this version. And When arktos changes version, mizar changes version too. Later after integration work of arktos and mizar, I don't hope we will hit issue because they are using different go version.

Reply:
I moved go install script from e2e test to bootstrap.sh.